src/api/layers/WMSCapabilitiesParser.class.js

```diff
@@ -245,21 +243,27 @@ export default class WMSCapabilitiesParser {
                     ignoreError
                 )
             ).filter((layer) => !!layer)
-            return new ExternalGroupOfLayers({
+            return new ExternalWMSLayer({
                 id: layerId,
                 name: title,
                 opacity,
                 visible,
                 baseUrl: url,
                 layers,
                 attributions,
+                wmsVersion: version,
+                format: 'png',
                 abstract,
                 extent,
                 legends,
                 isLoading: false,
                 availableProjections,
+                hasTooltip: queryable,
                 getFeatureInfoCapability: this.getFeatureInfoCapability(ignoreError),
                 currentYear,
+                customAttributes: params,
+                dimensions: dimensions,
+                timeConfig: this._getTimeConfig(layerId, dimensions),
             })
```

Comment:
this is exactly the same call to the constructor (plus the `layers` obviously) that outside the `if` block, might be better to add `layers` to the one outside and remove this return altogether